efiction/chapters.py

```diff
@@ -45,7 +46,7 @@ def __load_chapter_text_into_db(self, chapter_paths: List[dict]):
         :param chapter_paths: List of chapter metadata including path, author id and chapter id
```

Comment:
This comment needs to be updated:
```suggestion
        Load chapters text from the `stories` files into the chapters table. Checks for an encoding in the ini file and removes characters that are invalid in that encoding.
        :param chapter_paths: List of chapter metadata including path, author id and chapter id
```